src/utils/viewer/viewer.cpp

```diff
@@ -321,8 +332,8 @@ void Viewer::addObject(const std::string& configFile) {
   }
 
   Mn::Matrix4 T =
-      agentBodyNode_
-          ->MagnumObject::transformationMatrix();  // Relative to agent bodynode
+      agentBodyNode_->MagnumObject::transformationMatrix();  // Relative to
+                                                             // agent bodynode
```

Comment:
Looks like the linter made some unnecessary style changes several times in this file.